apps/docs/src/app/platform/component-docs/platform-forms/platform-form-container/platform-form-container-docs.component.html

```diff
@@ -43,15 +42,58 @@
 <separator></separator>
 
 <fd-docs-section-title id="column-binding" componentName="form-container"> Column Binding </fd-docs-section-title>
-<description>To bind column use the <code>[column]</code> attribute. The column attribute specify the respective layout for XL size, the rest automatically wrap according to layout pattern.</description>
+<description>To bind column use the <code>[column]</code> attribute. The column attribute specify the respective layout
+    for XL size, the rest automatically wrap according to layout pattern.</description>
 <component-example>
     <fdp-platform-form-basic-example></fdp-platform-form-basic-example>
 </component-example>
 <code-example [exampleFiles]="formContainerColumn"></code-example>
 
 <fd-docs-section-title id="group" componentName="form-container"> Form Field Group </fd-docs-section-title>
-<description>Wrap fields with the <code>[form-field-group]</code> component to group them. You can use <code>[column]</code> and <code>[rank]</code> attribute</description>
+<description>Wrap fields with the <code>[form-field-group]</code> component to group them. You can use
+    <code>[column]</code> and <code>[rank]</code> attribute
+</description>
 <component-example>
     <fdp-platform-form-group-example></fdp-platform-form-group-example>
 </component-example>
 <code-example [exampleFiles]="formContainerGroup"></code-example>
+
+<separator></separator>
+<fd-docs-section-title id="field-column-change" componentName="form-container"> Change of column number for a field
+</fd-docs-section-title>
+<description>To change the column number in form-group-container for a field while screen size changes, property
```

Comment:
You should run prettier on these new additions due to their strange formatting

Reply:
ran prettier